### PR TITLE
[executable.py] Code enhancements

### PR DESCRIPTION
**Pull request by veruu**
- make code more pythonic:
  - use `if x` or `if not x` instead of comparisons with 0 or None
  - `dict.get(key)` returns None in case key is not found so we can
    simplify the comparisons there as well
- add initialization of cfgurl - it is created conditionally so we'd get
  error in case the condition doesn't go through.
- get rid of `continue` statements

Signed-off-by: Veronika Kabatova <vkabatov@redhat.com>

### CODE REVIEW

File: skt/executable.py [hunk 1]
```diff
@@ -259,18 +261,18 @@ def cmd_run(cfg):
         save_state(cfg, {'baseretcode': baseres})
 
         # If baseline also fails - assume pass
-        if baseres != 0:
+        if not baseres:
```
Comment (spbnick) [inline skt/executable.py]: Should this be inverted?

File: skt/executable.py [hunk 2]
```diff
@@ -558,39 +560,35 @@ def load_config(args):
 
     if config.has_section('config'):
         for (name, value) in config.items('config'):
-            if name not in cfg or cfg.get(name) is None:
+            if not cfg.get(name):
                 cfg[name] = value
 
-    if config.has_section('publisher') and ('publisher' not in cfg or
-                                            cfg.get('publisher') is None):
+    if config.has_section('publisher') and not cfg.get('publisher'):
```
Comment (spbnick) [inline skt/executable.py]: Hmm, we'll need better name differentiation than `cfg` vs `config` here.

File: skt/executable.py (hunk 1 above)
Comment (veruu) [reply skt/executable.py]: Crap, good catch!

File: skt/executable.py (hunk 2 above)
Comment (veruu) [reply skt/executable.py]: Agreed, a lot of naming is confusing.

**Comment by spbnick** [comment]
Thanks you, @veruu, this looks good. Could you please rebase this before I merge it?

**Comment by veruu** [comment]
> Could you please rebase this before I merge it?

Done

**Comment by spbnick** [comment]
Thank you, Veronika, merged!

**Pull request merged**